rbac/feature_flags.py

```
@@ -182,5 +184,72 @@ def is_v2_edit_api_enabled(self, org_id: str) -> bool:
+            )
+
+            mode = variant.get("name", "kafka_active")
+
+            # Validate mode - only kafka_shadow and kafka_active are valid when flag is enabled
+            if mode == "kafka_shadow":
+                return "kafka_shadow"
+            else:
+                # Default to kafka_active for any other variant
+                return "kafka_active"
+
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)